packages/cubejs-api-gateway/src/LocalSubscriptionStore.ts

```diff
@@ -20,6 +20,7 @@ export class LocalSubscriptionStore {
     const connection = this.getConnection(connectionId);
     connection.subscriptions[subscriptionId] = {
       ...subscription,
+      cube: 'TODO',
```

Comment:
todo?

Reply:
ja, is nog niet helemaal klaar. ga hem vandaag test-klaar maken ;)